core/amber/.gitignore

```diff
@@ -12,6 +12,7 @@ project/plugins/project/
 .history
 .cache
 .lib/
+*.log
```

Comment:
shall we add this to the global `.gitignore` since the log files are mostly being written to `core/log` folder` for now. The exact repo structure will be changed in the future, only ignore logs in `core/amber` might not be enough. just a thought.

Reply:
done